src/controllers/event.js

```diff
@@ -1,16 +1,52 @@
-const createModal = require ( '../models/events.js');
-const {db} = require ('../databaseSetup');
-const {DataTypes} = require ('sequelize');
+const createModal = require('../models/events.js');
+const { db } = require('../databaseSetup');
+const { DataTypes } = require('sequelize');
 
 const Events = createModal(db, DataTypes);
 
 const getAllEvents = async (req, res) => {
-    try {
-        const events = await Events.findAll();
-        res.json(events);
-    } catch (error) {
-        res.json({ message: error.message });
-    }  
-}
+  const events = await Events.findAll();
+  res.json(events);
+};
+const getEventsById = async (req, res) => {
+  const events = await Events.findOne({
+    where: {
+      id: req.params.id,
+    },
+  });
+  res.json(events[0]);
```

Comment:
Does this endpoint was tested after last changes? I am not sure that it works properly.
FindOne method returns object instead of array. This is no 0 index for the object